msteams-platform/apps-in-teams-meetings/build-apps-for-teams-meeting-stage.md

```diff
@@ -304,6 +306,49 @@ The following code is an example of sharing app content to meeting stage view an
 * A sharing button is available on all meeting side panels for users with organizer or presenter roles.
 * Users can initiate sharing through a deep link or the Share in Meeting button.
 
+## Get app content in meeting side panel
+
+The `sharing.history.getContent` API enables you to fetch the content shared in a meeting and display in the meeting side panel.
+
+When the meeting starts, the app’s side panel automatically opens for the user who has shared content in the meeting. You can call the `sharing.history.getContent` API to fetch a list of shared content and display it in the side panel, making it readily accessible. For an upcoming recurring meeting, where the app is already added, developers can call the API to get the latest content to be displayed in the app's side panel.
+
+### Example
+
+```javascript
+// Define an async function
+async function fetchContentDetails() {
+    // Fetches a list of content details that was shared in the meeting
+    const contentDetails = await microsoftTeams.sharing.history.getContent();
+
+    // Above content details can be used to hydrate the meeting side panel 
+    // to share to meeting stage
+
+}
```

Comment:
Do we have sample code for this?

Reply:
@ShanmathiMayuramKrithivasan, can you please confirm if we have sample code for this? If so, can you please share?

Reply:
@Wajeed-msft 

There's no code sample for this, and the same is confirmed from Shanmathi. Here's the confirmation message from Shanmathi for your reference.

The code on line 321 is the same code for the new getContent API. Developers can use the data fetched from that call to hydrate the side panel - there is no sample code for this as developers can use it in any way they can. This is not relevant to the API.